### PR TITLE
Adding new vagrant box

### PR DESCRIPTION
**Pull request by sergio-ocon**
Changed the description of the vagrant box to the official vagrant box

### CODE REVIEW

File: site/docs/get-started/vagrant.md [hunk 1]
```diff
@@ -19,10 +19,10 @@ Additional details on how to configure the system properly can be found in the l
 
 ### Step 2. Download and deploy the appliance
 
-1. Pull the ManageIQ Vagrant box from [Atlas](https://atlas.hashicorp.com/vagrant) and run it
+1. Pull the ManageIQ Vagrant box from [Atlas](https://atlas.hashicorp.com/ManageIQ) and run it
```
Comment (geertj) [inline site/docs/get-started/vagrant.md]: Also convention seems to be to use lower case on Atlas. That would be easier to type.

Comment (sergio-ocon) [inline site/docs/get-started/vagrant.md]: I copied and pasted it from Atlas directly.
Using ManageIQ there gives the full list of boxes under ManageIQ.
But it works with lower cases, I will update it tomorrow

Comment (sergio-ocon) [inline site/docs/get-started/vagrant.md]: updated so now is lowercase. The actual vagrant init has to be as it is or it won't work (it complains saying the the  actual name has some uppercase and I am trying to download it lowercase)

File: site/docs/get-started/vagrant.md [hunk 2]
```diff
@@ -19,10 +19,10 @@ Additional details on how to configure the system properly can be found in the l
 
 ### Step 2. Download and deploy the appliance
 
-1. Pull the ManageIQ Vagrant box from [Atlas](https://atlas.hashicorp.com/vagrant) and run it
+1. Pull the ManageIQ Vagrant box from [Atlas](https://atlas.hashicorp.com/manageiq) and run it
 ```bash
     $ mkdir manageiq-vagrant; cd manageiq-vagrant
-    $ vagrant init geertj/manageiq
+    $ vagrant init ManageIQ/darga
```
Comment (sergio-ocon) [inline site/docs/get-started/vagrant.md]: you can't use manageiq/darga here

File: site/docs/get-started/vagrant.md (hunk 1 above)
Comment (Fryguy) [inline site/docs/get-started/vagrant.md]: I changed the vagrant boxes to be all lowercase now (Had to change the org name).  So you can update it to lowercase here.

File: site/docs/get-started/vagrant.md (hunk 2 above)
Comment (Fryguy) [inline site/docs/get-started/vagrant.md]: Nope.  I think it should be driven off the .yml file.

**Comment by hayesr** [comment]
Is this ready to go?

**Comment by sergio-ocon** [comment]
Ready to go now. Changed to full lowercase after Jason did the change in atlas

**Pull request merged**